articles/libraries/auth0js/v8/migration-guide.md

```diff
@@ -163,11 +163,13 @@ webAuth.passwordlessVerify({
 );
 ```
 
-## The parseHash Method
+## Id_token Validation
 
-The `parseHash` method now validates the id_token. In order for this to work properly, the token should be signed using RS256, and will fail if the token is signed using HS256.
+When the `id_token` signature method is HS256, auth0.js will call the [/userinfo](/api/authentication#get-user-info) endpoint to retrieve user information and populate the idTokenPayload that is send to the 'parseHash' callback function.
 
-This can be avoided by either switching how your id_tokens are signed, or by manually parsing hashes, rather than using the `parseHash` method.
+If the `id_token` is signed with RS256, auth0.js will validate the token, decode it, and populate the idTokenPayload with the decoded data.
+
+We recommend that you use RS256 for signing tokens in Single Page Applications.
```

Comment:
Backticks on the variable, and maybe the next sentence as a note to make it stand out a bit more?

If the `id_token` is signed with RS256, auth0.js will validate the token, decode it, and populate the `idTokenPayload` with the decoded data.

::: note 
It is recommended that RS256 is used to sign tokens in single page applications.
:::